### PR TITLE
Feature/update model access doc page

### PR DESCRIPTION
**Pull request by cvlsk**
## What are you changing in this pull request and why?

The example code blocks to show how a model's `access: public` is defined was not accurate. Updating the relevant sections to clearly show how to set a model's access to public in 
1) the `properties.yml` file (either directly or in config block (dbt 1.7 onwards)), 
2) in `dbt_project.yml` file and 
3) in the config section of the `model.sql` file

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/reference/resource-configs/access.md [hunk 1]
```diff
@@ -23,38 +23,55 @@ Access modifiers may be applied to models one-by-one in YAML properties. In v1.5
 
 <VersionBlock firstVersion="1.7">
 
-You can apply access modifiers in config files, including `the dbt_project.yml`, or to models one-by-one in YAML properties. Applying access configs to a subfolder modifies the default for all models in that subfolder, so make sure you intend for this behavior. When setting individual model access, a group or subfolder might contain a variety of access levels, so when you designate a model with `access: public` make sure you intend for this behavior.
+You can apply access modifiers in config files, including the `dbt_project.yml`, or to models one-by-one in `properties.yml`. Applying access configs to a subfolder modifies the default for all models in that subfolder, so make sure you intend for this behavior. When setting individual model access, a group or subfolder might contain a variety of access levels, so when you designate a model with `access: public` make sure you intend for this behavior.
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/access.md:26]: great catch!

Comment (cvlsk) [reply website/docs/reference/resource-configs/access.md:26]: thanks! can you approve/merge this then? @mirnawong1

Comment (mirnawong1) [reply website/docs/reference/resource-configs/access.md:26]: thanks @cvlsk ! will review this for you now

File: website/docs/reference/resource-configs/access.md [hunk 2]
```diff
@@ -23,38 +23,55 @@ Access modifiers may be applied to models one-by-one in YAML properties. In v1.5
...
 
 There are multiple approaches to configuring access:
 
-In the model configs of `dbt_project.yml`: 
+In `properties.yml`: 
+
+<File name='models/<properties_my_public_model>.yml'>
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/access.md]:
```suggestion
<File name='models/properties_my_public_model.yml'>
```

File: website/docs/reference/resource-configs/access.md [hunk 3]
```diff
@@ -23,38 +23,55 @@ Access modifiers may be applied to models one-by-one in YAML properties. In v1.5
...
   - name: my_public_model
     access: public # Older method, still supported
     
 ```
+</File>
+  
 Or (but not both)
 
-```yaml
+<File name='models/<properties_my_public_model>.yml'>
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/access.md]:
```suggestion
<File name='models/properties_my_public_model.yml'>
```

**Review by mirnawong1 — APPROVED** [review]
🚢 lgtm! thank you so much for opening this up @cvlsk - we really appreciate your initiative and opening this pr to make the docs clearer!

File: website/docs/reference/resource-configs/access.md [hunk 4]
```diff
@@ -23,46 +23,65 @@ Access modifiers may be applied to models one-by-one in YAML properties. In v1.5
...
+  version: 2
+  
+  models:
+    - name: my_public_model
+      access: public # Older method, still supported
+      
+  ```
+  </File>
+  
+- Newer method (v1.7 or older). Use either the older method or the new method, but not both for the same model:
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/access.md]:
```suggestion
- In `properties.yml` using the new method (for v1.7 or higher). Use either the older method or the new method, but not both for the same model:
```

**Pull request merged**